src/backend/base/langflow/services/database/service.py

```diff
@@ -136,7 +136,14 @@ def on_connection(self, dbapi_connection, _connection_record) -> None:
     @asynccontextmanager
     async def with_session(self):
         async with AsyncSession(self.engine, expire_on_commit=False) as session:
-            yield session
+            try:
+                yield session
+                if session.is_active:
+                    await session.commit()
+            except Exception:
+                logger.exception("An error occurred during the session scope.")
```

Comment:
logger.exception will flood logs.
```suggestion
                logger.error("An error occurred during the session scope.")
```